Plutarch/Api/V1.hs

```diff
@@ -421,8 +433,11 @@ instance PIsDataRepr PTxId where
 newtype PTxOutRef (s :: S)
   = PTxOutRef (Term s (PDataList '[PTxId, PInteger]))
   deriving
-    (PMatch, PIsData, PUnsafeLiftDecl Plutus.TxOutRef)
-    via PIsDataReprInstances PTxOutRef Plutus.TxOutRef
+    (PMatch, PIsData)
+    via PIsDataReprInstances PTxOutRef
+
+instance PUnsafeLiftDecl PStakeValidatorHash where type PLifted PStakeValidatorHash = Plutus.StakeValidatorHash
+deriving via (DerivePConstantViaData Plutus.StakeValidatorHash PStakeValidatorHash) instance (PConstant Plutus.StakeValidatorHash)
```

Comment:
Looks like some of these instance decls got pasted in the wrong part of the file :sweat_smile:

Reply:
lol